prebuilt/common/bin/chrootinit

```diff
@@ -2,41 +2,43 @@
 # Initial chroot setup
 
 if [ ! -x /system/bin/getprop ]; then
-  printf "Unable to execute /system/bin/getprop (needed for device detection)\n"
+  /system/bin/toybox printf "Unable to execute /system/bin/getprop (needed for device detection)\n"
   exit 1
 fi
 
-if grep -q squashfs /proc/filesystems; then
+if /system/bin/toybox grep -q squashfs /proc/filesystems; then
   if [ ! -f /sdcard/Android/data/com.pwnieexpress.android.pxinstaller/files/stockchroot.sfs ]; then
-    printf "Unable to read stockchroot.sfs\n"
+    /system/bin/toybox printf "Unable to read stockchroot.sfs\n"
     exit 1
   else
     chroot_file="/sdcard/Android/data/com.pwnieexpress.android.pxinstaller/files/stockchroot.sfs"
   fi
 else
-  printf "No squashfs support present\n"
+  /system/bin/toybox printf "No squashfs support present\n"
   exit 1
 fi
 
 # Error fast
 set -e
 
 # Unpack chroot
-busybox mkdir -p /data/local/kali
-busybox mkdir -p /data/local/kali-tmp
-busybox mount -t auto "${chroot_file}" /data/local/kali-tmp
-busybox cp -a /data/local/kali-tmp/* /data/local/kali
-busybox umount /data/local/kali-tmp
-busybox rm -rf /data/local/kali-tmp
+/system/bin/toybox mkdir -p /data/local/kali
+/system/bin/toybox mkdir -p /data/local/kali-tmp
+/system/bin/toybox cp "${chroot_file}" /data/local
+/system/bin/toybox mount -t auto /data/local/stockchroot.sfs /data/local/kali-tmp
+/system/bin/toybox cp -a /data/local/kali-tmp/* /data/local/kali
+/system/bin/toybox umount /data/local/kali-tmp
+/system/bin/toybox rm -rf /data/local/kali-tmp
+/system/bin/toybox rm /data/local/stockchroot.sfs
 
 # Fix chroot permissions
-busybox chmod 0755 /data/local/kali
+/system/bin/toybox chmod 0755 /data/local/kali
 
 # Detect form factor
-hardw=$(/system/bin/getprop ro.build.characteristics | busybox cut -d, -f 1)
+hardw=$(/system/bin/getprop ro.build.characteristics |/system/bin/toybox cut -d, -f1)
 case ${hardw} in
   phone)
-    PRODUCT="Pwn Phone 2014"
+    PRODUCT="Pwn Phone"
```

Comment:
I was going to create a separate value set to "aopp" (and maybe a system prop), but I'm fine setting that here. Can it be "aopp" (lower case) though?